### PR TITLE
clean up UT test

### PR DESCRIPTION
**Pull request by sailorlqh**

### CODE REVIEW

**Review by sethvargo — APPROVED** [review]

**Comment by capri-xiyue** [comment]
UT of logging controller is just an example of https://github.com/abcxyz/lumberjack/issues/392.
https://github.com/abcxyz/lumberjack/issues/392 needs to clean all the UTs in lumberjack java code

**Review by capri-xiyue — CHANGES_REQUESTED** [review]
https://github.com/abcxyz/lumberjack/issues/392 needs to clean up all lumberjack java UT tests. Is this part of the fix of https://github.com/abcxyz/lumberjack/issues/392 and you will create other PRs to clean up other lumberjack java UT tests?

File: clients/java-logger/shell/src/test/java/com/abcxyz/lumberjack/loggingshell/LoggingControllerTest.java [hunk 1]
```diff
@@ -84,41 +84,23 @@ void loggingWithTraceIdReturnsOk() throws Exception {
...
-    mockMvc.perform(GET_REQUEST_BUILDER_WITH_EMAIL_AND_TRACE_ID_AND_JVS_TOKEN);
-    verify(loggingClient).log(auditLogRequestCaptor.capture());
-    assertThat(auditLogRequestCaptor.getValue().getTimestamp()).isEqualTo(TEST_TIMESTAMP);
+  void loggingWithoutInterceptedEmailCausesBadRequestError() throws Exception {
+    mockMvc.perform(GET_REQUEST_BUILDER).andExpect(status().isBadRequest());
   }
 
   @Test
-  void loggingUsesTheEmailInRequestAuthenticationInfo() throws Exception {
+  void loggingWithCustomInfoAndAssertResult() throws Exception {
```
Comment (capri-xiyue) [inline clients/java-logger/shell/src/test/java/com/abcxyz/lumberjack/loggingshell/LoggingControllerTest.java:92]: Can we merge `loggingWithTraceIdReturnsOk() throws Exception` in this test case?

File: clients/java-logger/shell/src/test/java/com/abcxyz/lumberjack/loggingshell/LoggingControllerTest.java [hunk 2]
```diff
@@ -77,48 +77,23 @@ void loggingWithoutTraceIdCausesBadRequestError() throws Exception {
   }
 
   @Test
-  void loggingWithTraceIdReturnsOk() throws Exception {
-    mockMvc
-        .perform(GET_REQUEST_BUILDER_WITH_EMAIL_AND_TRACE_ID_AND_JVS_TOKEN)
-        .andExpect(status().isOk());
```
Comment (capri-xiyue) [inline clients/java-logger/shell/src/test/java/com/abcxyz/lumberjack/loggingshell/LoggingControllerTest.java:83]: When you remove this one, did you verify it in other test cases that the status is ok?

Comment (sailorlqh) [reply clients/java-logger/shell/src/test/java/com/abcxyz/lumberjack/loggingshell/LoggingControllerTest.java:83]: I think the removed thing is duplicated to the one in line86, so if `loggingWithTraceIdAndCustomInfoAndAssertResult` passes, the status will also be checked.

**Review by capri-xiyue — APPROVED** [review]

**Comment by capri-xiyue** [comment]
To merge this PR, you need all commits to be signed. You can squash all the previous commits and force push one signed commit.

**Pull request merged**